source/PlayState.hx

```diff
@@ -7,6 +7,7 @@ class PlayState extends FlxState
 	override public function create()
 	{
 		super.create();
+		var ourHero:Hero = new Hero(300, 300, "John");
```

Comment:
Don't forget that you need to call the `PlayState`'s `add()` method to make your instantiated `Hero` object actually show up on screen. 

The assignment specifies that you should see the "hero" on-screen when you're done, so make sure to re-read the instructions carefully.